app/models/course/assessment/assessment_ability.rb

```diff
@@ -25,7 +25,7 @@ def assessment_course_staff_hash
 
   def submission_attempting_hash(user)
     { workflow_state: 'attempting' }.tap do |result|
-      result.reverse_merge(course_user: { user_id: user.id }) if user
+      result.reverse_merge(experience_points_record: { course_user: { user_id: user.id } }) if user
```

Comment:
It doesn't work for `course_staff`: it seems like rails is trying to find the `course_user_id` column on `course_assessment_submission`. Took me a while to find this out, might be better to be more explicit.